debugging-via-dotnet-dump/test.sh

```diff
@@ -319,18 +319,6 @@ grep 'Attributes: *PEFile' dump.out
 grep 'MetaData start address: *0' dump.out
 
 
-# FIXME: dumpmt seems to work on 7.0.1 but not 7.0.0?
-# heading "dumpmt"
-# dump-analyze 'name2ee *!System.String' > dump.out
-# cat dump.out
-# grep 'MethodTable:' dump.out | awk '{print $2}'
-# string_method_table=$(grep 'MethodTable:' dump.out | awk '{print $2}')
-# dump-analyze "dumpmt ${string_method_table}" > dump.out
-# cat dump.out
-# grep -E '^Name:[ \n\t]+System.String' dump.out
-# grep -E "^File:[ \n\t]+${framework_dir}" dump.out
-
-
```

Comment:
Any chance we can review and then enable this for 8.0 and later?

Reply:
I enabled it and tested for .NET 6 and 8, seems to be working correctly.

Reply:
Okay, this seems to be failing. My slight preference is to leave it commented out (like it was before this PR) so we can investigate it separately in the future. But I am okay if you want to remove it too.